### DISCUSSION TITLE
Add web color names to color format options for accessibility

### DISCUSSION THREAD

**Discussion by attokojeff** [jaames/iro.js#169] [Feature Requests]
Great tool we're using to teach students about hex and rgb values.  However to take it a step closer to meet accessibility guidelines for color blindness it would be great if web color names could also be supported as a color format.  Obviously can't name every single value, but would still be useful.  Color names reference: https://www.w3schools.com/colors/colors_hex.asp.  Thanks!

**Comment by jaames** [comment]
I've considered this in the past, but felt I couldn't really justify the extra bundle size needed for an adequate list of color names, especially considering that English names might not even be useful for a lot of people's use-cases depending on what audiences they're catering to.

Maybe it would be better to provide some code samples for incorporating https://github.com/dtao/nearest-color along with a nice established color list like https://github.com/meodai/color-names?

**Comment by attokojeff** [comment]
That makes sense.  Very interesting links.  My skills are still budding but I may look into incorporating those later.  Thanks so much for the quick response!
